openfisca_core/legislations.py

```diff
@@ -48,7 +48,9 @@ def combine_tax_scales(self):
         """Combine all the MarginalRateTaxScales in the node into a single MarginalRateTaxScale."""
         combined_tax_scales = None
         for name, child in self.iteritems():
-            assert isinstance(child, taxscales.AbstractTaxScale), child
+            if not isinstance(child, taxscales.AbstractTaxScale):
+                log.info('Skipping {} with value {} because it is not a tax scale'.format(name, child))
```

Comment:
Please log unicode strings only (add the "u") :)

By the way is it really an info or is it an error, with a better error message?